### PR TITLE
Itdev34822 selectable pi

### PR DESCRIPTION
**Pull request by harterjwustl**

### CODE REVIEW

File: coldfront/core/project/forms.py [hunk 1]
```diff
@@ -18,6 +18,17 @@
     'EMAIL_DIRECTOR_EMAIL_ADDRESS', '')
 
 
+
+class ProjectCreateForm(forms.ModelForm):
+    class Meta:
+        model = Project
+        fields = ['title', 'pi', 'description', 'field_of_science']
+        
+    pi = PrincipalInvestigatorField(
```
Comment (melonistic) [inline coldfront/core/project/forms.py:27]: Although `pi` is a well-known term, `principal_investigator` is more accessible and easier to maintain than `pi.`

This is just a recommendation.

Comment (harterjwustl) [reply coldfront/core/project/forms.py:27]: Good call out

Comment (harterjwustl) [reply coldfront/core/project/forms.py:28]: Actually, that name is bound by the name of the field in the build in coldfront model, so not something we can really change

File: coldfront/core/project/fields.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+from coldfront_plugin_qumulo.validators import validate_single_ad_user
+
+
+class PrincipalInvestigatorField(forms.CharField):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+    
+    def to_python(self, value):
+        return User.objects.get_or_create(username=value)[0]
```
Comment (melonistic) [inline coldfront/core/project/fields.py:14]: Note to self: get_or_create returns a tuple, where the first element of the tuple is the object.
user_obj, created = User.objects.get_or_create(username=value)

**Review by peaches7083 — APPROVED** [review]
LGTM

**Review by melonistic — APPROVED** [review]

**Comment by harterjwustl** [comment]
> LGTM

I had to google this and best guess is:

Looks Good To Me?

**Pull request merged**